2020-02-Spring/2_linked_lists/README.md

```diff
@@ -158,29 +197,177 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
+### 3. Reorder List Solution
 
-Source: TODO :bug:
+<details>
+<summary>Click to see solutions and solution explanation</summary>
 
-#### Naive/Simple Solution 
+#### Algorithm Overview
 
-<details>
-<summary>Click to see solution</summary>
+This problem is a combination of three common linked list manipulations:
 
-TODO put your solution here :bug:
+* Finding the middle of a linked list
+* Reversing a linked list
+* Merging two linked lists
 
-</details>
```

Comment:
Something is funky about the way this got rearranged -- I think a closing `</details>` tag is missing. I've added one at the very bottom underneath your tests heading.
![funky](https://user-images.githubusercontent.com/29528527/80653718-9f5d2880-8a2f-11ea-885f-30fee310a3b8.png)